controller/controller_server/test_settings.py

```diff
@@ -4,6 +4,7 @@
 pool = "pool1"
 space_efficiency = "thin"
 volume_name = "volume"
+object_id = "0"
```

Comment:
consider using more indicative name and value, e.g.
```suggestion
internal_id = "internal-id"
```

Reply:
done

Reply:
same for [here](https://github.com/IBM/ibm-block-csi-driver/blob/aeb2553d0681192f370ea1520186d38572486976/controller/tests/array_action/xiv/array_mediator_xiv_tests.py#L75), [here](https://github.com/IBM/ibm-block-csi-driver/blob/aeb2553d0681192f370ea1520186d38572486976/controller/tests/array_action/svc/array_mediator_svc_test.py#L273), etc...

Reply:
those are storage objects id parameter, e.g. `cli_volume.id`

Reply:
oh, right

Reply:
then how about:
```suggestion
object_id = "object-id"
```

Reply:
`Shadows built-in name 'id' `
done `object_id`..

Reply:
sorry, I meant `'id': 'test-id'`

Reply:
ok, done..